modules/core/keeper/msg_server.go

```diff
@@ -164,29 +113,14 @@ func (k Keeper) ConnectionOpenTry(goCtx context.Context, msg *connectiontypes.Ms
 		return nil, err
 	}
 
-	connectionID, err := k.ConnectionKeeper.ConnOpenTry(
+	if _, err := k.ConnectionKeeper.ConnOpenTry(
```

Comment:
Same question as with the `clientID` returned by `CreateClient`.

Reply:
Same answer as above